src/posts/introducing-xwrf/index.md

```diff
@@ -0,0 +1,381 @@
+---
+title: 'Making WRF model output Xarray-friendly using xWRF'
+date: '2022-10-12'
+authors:
+  - name: Lukas Pilz
+    github: lpilz
+  - name: Jonathan (JT) Thielen
+    github: jthielen
+---
+
+![xWRF logo](https://raw.githubusercontent.com/xarray-contrib/xwrf/main/docs/source/_static/xwrf_logo_bg_light.svg)
+
+## TLDR
```

Comment:
Can you put the full term here? For those who do not know TLDR?

Reply:
SGTM ;)

Reply:
Or summary/overview works :)